requirements.test.txt

```diff
@@ -9,3 +9,6 @@ pytest-cov
 codecov
 rtree>=0.8
 pysal
+geoalchemy2
+boto3
+fiona==1.7.12 # Fixing version to avoid conda installing alpha #853
```

Comment:
I don't follow the explanatory comment, and would prefer not to be pinning a specific version here if not absolutely necessary.

Reply:
I totally agree there, there just was a build error at the time due to fiona I think it should be solved by now